etc/requirements.txt

```diff
@@ -82,7 +82,7 @@ empyrical==0.5.0
 tables==3.4.3
 
 # For trading calendars
-trading-calendars==1.4.0
+trading-calendars==1.4.1
```

Comment:
Once we cut another micro release/update this, you'll also need to update `conda/trading-calendars/meta.yaml` like seen in https://github.com/quantopian/zipline/pull/2341/files